.github/workflows/tests.yaml

```
@@ -0,0 +1,34 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [7.3, 7.4, 8.0]
+        laravel: [6.*, 7.*, 8.*]
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }}
```

```suggestion
    name: PHP ${{ matrix.php }} - Laravel ${{ matrix.laravel }}
```

Being a bit more verbose here.